### ISSUE TITLE
md-card if used with ng-repeat doesn't refresh if the model changes

### ISSUE THREAD

**Issue by venkateshmanohar** [angular/material#6540]
`<div class='md-padding' layout="column">
	<md-card ng-repeat="user in users " ng-drag="true" ng-drag-data="user" data-allow-transform="true">
	<md-card-content>
	<div >
		<h2>{{user.name}}</h2>
		<div>{{user.age}}</div>
		<div>{{user.retiredate}}</div>
		<div>{{user.city}}</div>
	</div>
	</md-card-content> </md-card>
</div>`

I am basically iterating over the list of users to display their name , age and city etc in a card. If i add/remove the user dynamically from the model $scope.users then the list of md-cards are reflecting the latest changes in the model and it will still be showing the same list which will be displayed on launch.

angular.module('MyApp', ['ngMaterial', 'ngDraggable'])
  .controller('AppCtrl', function($scope) {
    $scope.users =[
                   {name:"George", age:32, retiredate:"March 12, 2014", city:"Bangalore"},
                   {name:"Edward", age:17, retiredate:"June 2, 2023", city:"Mumbai"},
                   {name:"Christine", age:58, retiredate:"December 20, 2036", city:"Delhi"},
                   {name:"Sarah", age:62, retiredate:"April 30, 2020", city:"Chennai"},
                   {name:"George1", age:32, retiredate:"March 12, 2014", city:"Bangalore"},
                   {name:"Edward1", age:17, retiredate:"June 2, 2023", city:"Mumbai"},
                   {name:"Christine1", age:58, retiredate:"December 20, 2036", city:"Delhi"},
                   {name:"Sarah1", age:62, retiredate:"April 30, 2020", city:"Chennai"},
                   {name:"George2", age:32, retiredate:"March 12, 2014", city:"Bangalore"},
                   {name:"Edward2", age:17, retiredate:"June 2, 2023", city:"Mumbai"},
                   {name:"Christine2", age:58, retiredate:"December 20, 2036", city:"Delhi"},
                   {name:"Sarah2", age:62, retiredate:"April 30, 2020", city:"Chennai"}
                 ];

// EVEN AFTER EXECUTING THE BELOW CODE THE MD-CARD WONT DISPLAY LATEST USERS.
$scope.onDragSuccess1=function(data,evt){
    console.log("133","$scope","onDragSuccess1", "", evt);
    var index = $scope.users.indexOf(data);
    if (index > -1) {
        $scope.users.splice(index, 1);
    }
}

})
;

**Comment by venkateshmanohar** [comment]
<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-animate.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-aria.min.js"></script>
<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular-messages.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/angular_material/0.9.4/angular-material.js"></script>

These particular version of the files solved the issue.

**Comment by wcjr** [comment]
@topherfangio , I'm noticing this behavior again.  I wanted to check in with you before I tried to setup a plunkr demo for it.  Do you happen to remember what fixed this and could that have been reverted somehow in the latest release?  (I'm currently using ng 1.5.8 and md 1.1.1).  Thanks!

@venkateshmanohar have you noticed this returning on later versions of MD?

**Comment by topherfangio** [comment]
@wcjr Unfortunately, I do not think I really ever investigated this issue since it was closed by the original author.

I think putting together a new issue with a Codepen would be best. Feel free to ping me on it and I can take a look.

**Comment by VenkateshManohar27** [comment]
@wcjr I have stopped using angular materials and i was basically doing a POC when i had raised this issue. But the issue was resolved by using the specific set of javascript libararies as mentioned in my previous comment. I have also lost the code and hence will not be able to give more information on this issue.

**Comment by wcjr** [comment]
ok, thanks guys.  I'll put together a simple example for it and open a new issue.  I'll make sure to reference it from here.

**Comment by wcjr** [comment]
Well, I'm not able to reproduce this in a simplified implementation so I must have some unique combination of layout options in place that is causing this issue for me.  Thanks for being so responsive, hope you all have a great weekend.
